website/content/docs/ecs/get-started/install.mdx

```diff
@@ -72,136 +70,6 @@ resource "aws_ecs_service" "my_task" {
-  type                     = "ingress"
-  from_port                = 8500
-  to_port                  = 8500
-  protocol                 = "tcp"
-  source_security_group_id = module.dev_consul_server.lb_security_group_id
-  security_group_id        = data.aws_security_group.vpc_default.id
-}
-
-output "consul_server_url" {
-  value = "http://${module.dev_consul_server.lb_dns_name}:8500"
-}
-```
-
--> **NOTE:** The documentation for all possible inputs can be found in the [module reference
-docs](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/dev-server?tab=inputs).
-
-The example code above will create a Consul server ECS task and Application Load
-Balancer for the Consul UI. You can then use the output `consul_server_url` as
-the URL to the Consul server.
-
-## Task IAM Role
-
-Your tasks must have an IAM role that allows them to list and describe
-other tasks. This is required in order for the tasks to find the IP
-address of the Consul server.
-
-The specific permissions needed are:
-
-1. `ecs:ListTasks` on resource `*`.
-1. `ecs:DescribeTasks` on all tasks in this account and region. You can either
-   use `*` for simplicity or scope it to the region and account, e.g. `arn:aws:ecs:us-east-1:1111111111111:task/*`. If
-   your account is configured to use the new, [longer ECS task ARN format](https://docs.aws.amazon.com/AmazonECS/latest/userguide/ecs-account-settings.html#ecs-resource-ids)
-   then you can further scope `ecs:DescribeTasks` down to tasks in a specific cluster, e.g. `arn:aws:ecs:us-east-1:1111111111111:task/MY_CLUSTER_NAME/*`.
-
-The IAM role's ARN will be passed into the `mesh-task` module in the next step
-via the `task_role_arn` input.
-
--> **NOTE:** There are two IAM roles needed by ECS Tasks: Execution roles and
-Task roles. Here we are referring to the Task role, not the Execution role.
-The Execution role is used by ECS itself whereas the Task role defines the
-permissions for the containers running in the task.
-
-Terraform for creating the IAM role might look like:
-
-```hcl
-data "aws_caller_identity" "this" {}
-
-resource "aws_iam_role" "this_task" {
-  name = "this_task"
-  assume_role_policy = jsonencode({
-    Version = "2012-10-17"
-    Statement = [
-      {
-        Action = "sts:AssumeRole"
-        Effect = "Allow"
-        Sid    = ""
-        Principal = {
-          Service = "ecs-tasks.amazonaws.com"
-        }
-      },
-    ]
-  })
-
-  inline_policy {
-    name = "this_task"
-    policy = jsonencode({
-      Version = "2012-10-17"
-      Statement = [
-        {
-          Effect = "Allow"
-          Action = [
-            "ecs:ListTasks",
-          ]
-          Resource = "*"
-        },
-        {
-          Effect = "Allow"
-          Action = [
-            "ecs:DescribeTasks"
-          ]
-          Resource = [
-            "arn:aws:ecs:${var.region}:${data.aws_caller_identity.this.account_id}:task/*",
-          ]
-        }
-      ]
-    })
-  }
-}
-
-```
-
```

Comment:
we also need to update code snippet for `my_task` below. It needs:
1. Remove `execution_role_arn` and `task_role_arn`
2. Change `consul_server_service_name` to `retry_join`: `retry_join = "<address of the Consul server>"`